app/styles/main.scss

```diff
@@ -279,9 +287,15 @@ body {
         background-color: $headingBgColor;
         height: $header-height;
 
+        i, img {
+            margin-left: 10px;
+            margin-right: 10px;
+        }
+
         h1, h2, h3, h4, h5 {
             color: $headingTxtColor;
-            letter-spacing: -1px;
+            margin-top: 10px;
+            margin-bottom: 10px;
```

Comment:
are these margins needed? there is static space and text can be expanded to several lines.